config/prow/plugins.yaml

```diff
@@ -160,6 +160,12 @@ blockades:
   blockregexps:
   - ^pkg/util/[^/]+\.go$
   explanation: "Utility code must be added to a subpackage like pkg/util/flag, not pkg/util directly. See: #49923"
+- repos:
+  - kubernetes/kubernetes
+  blockregexps:
+  - ^test/conformance/testdata/
```

Comment:
@nikhita let's specifically deal with `conformance.yaml`

(as we will run into problems updating OWNERS or deleting bazel if/when we get to it)

Reply:
thanks for taking care of this!

Reply:
Done!